python-bindings/overview_article/ctypes_test.py

```diff
@@ -2,12 +2,13 @@
 """ Simple examples of calling C functions through ctypes module. """
 import ctypes
 import pathlib
-
+import sys
 
 if __name__ == "__main__":
     # Load the shared library into c types.
-    libname = pathlib.Path().absolute() / "libcmult.so"
-    c_lib = ctypes.CDLL(libname)
+    lib_ext = ".dll" if sys.platform.startswith("win") else ".so"
+    libname = pathlib.Path() / f"cmult{lib_ext}"
+    c_lib = ctypes.CDLL(libname.resolve().__str__())
```

Comment:
Linux and mac users are *really* gong to expect `libcmult.so`.  I'd recommend doing this instead of line 9:

```python
if sys.platform.startswith("win"):
    libname = "cmult.dll"
else:
    libname = "libcmult.so"
```
and then just using `libname` instead of the fstring on line 10.